pastis/temporal_analysis/hi_modes.ipynb

```diff
@@ -0,0 +1,1025 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "169af060",
+   "metadata": {},
+   "source": [
+    "## This notebook contains e2e analysis used to allocate tolerances for each $\\color{red}{\\text{High Spatial Frequency}}$ mode for a segmented telescope."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3da09bfa",
+   "metadata": {},
+   "source": [
+    "### Import necessary libraries and scripts"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d37e0e51",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir(\"/Users/asahoo/repos/PASTIS\")\n",
+    "import time\n",
+    "from shutil import copy\n",
+    "from astropy.io import fits\n",
+    "import astropy.units as u\n",
+    "import hcipy\n",
+    "import numpy as np\n",
+    "import pastis.util as util    \n",
+    "from pastis.config import CONFIG_PASTIS \n",
+    "from pastis.e2e_simulators.luvoir_imaging import LuvoirA_APLC \n",
+    "from pastis.e2e_simulators.generic_segmented_telescopes import SegmentedAPLC\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "from scipy.interpolate import griddata\n",
+    "import exoscene.image\n",
+    "import exoscene.star\n",
+    "import exoscene.planet\n",
+    "from exoscene.planet import Planet\n",
+    "from astropy.io import fits as pf\n",
+    "from pastis.analytical_pastis.temporal_analysis import req_closedloop_calc_batch\n",
+    "from matplotlib.colors import TwoSlopeNorm\n",
+    "import matplotlib.ticker as ticker"
```

Comment:
as above

Reply:
the imports are still all over the place, but I guess we don't care at this point